code/socialDistribution/urls.py

```diff
@@ -4,20 +4,20 @@
 app_name = 'socialDistribution'
 urlpatterns = [
   path('', views.index, name='index'),
-  path('login/', views.loginPage, name='login'),
-  path('register/', views.register, name='register'),
-  path('logout/', views.logoutUser, name='logout'),
-  path('author/', views.authors, name='authors'),
-  path('author/<int:author_id>/', views.author, name='author'),
-  path('author/<int:author_id>/home/', views.home, name='home'),
-  path('author/<int:author_id>/posts/', views.posts, name='posts'),
-  path('author/<int:author_id>/befriend/', views.befriend, name='befriend'),
-  path('author/<int:author_id>/un-befriend/', views.un_befriend, name='un-befriend'),
-  path('<int:author_id>/accept-friend/', views.accept_friend, name='accept-friend'),
-  path('create/', views.create, name='create'),
-  path('profile/', views.profile, name='profile'),
-  path('user/', views.user, name='user'),
-  path('like-post/<int:id>', views.likePost, name='likePost'),
-  path('delete-post/<int:id>', views.deletePost, name='deletePost'),
-  path('edit-post/<int:id>', views.editPost, name='editPost'),
+  path('app/login/', views.loginPage, name='login'),
+  path('app/register/', views.register, name='register'),
+  path('app/logout/', views.logoutUser, name='logout'),
+  path('app/author/', views.authors, name='authors'),
+  path('app/author/<int:author_id>/', views.author, name='author'),
+  path('app/author/<int:author_id>/home/', views.home, name='home'),
+  path('app/author/<int:author_id>/posts/', views.posts, name='posts'),
+  path('app/author/<int:author_id>/befriend/', views.befriend, name='befriend'),
+  path('app/author/<int:author_id>/un-befriend/', views.un_befriend, name='un-befriend'),
+  path('app/<int:author_id>/accept-friend/', views.accept_friend, name='accept-friend'),
+  path('app/create/', views.create, name='create'),
+  path('app/profile/', views.profile, name='profile'),
+  path('app/user/', views.user, name='user'),
+  path('app/like-post/<int:id>', views.likePost, name='likePost'),
+  path('app/delete-post/<int:id>', views.deletePost, name='deletePost'),
+  path('app/edit-post/<int:id>', views.editPost, name='editPost'),
```

Comment:
Instead of hardcoding app/ everywhere here, can we add the app/ prefix to cmput404/urls.py? 

For example, `path('app/', include('socialDistribution.urls'))`

Reply:
The API paths won't be given the "app/" prefix if you do this since all the API URLs are defined in api.urls.

```python
urlpatterns = [
    path('', include('socialDistribution.urls')),
    path('api/', include('api.urls')),
    path('admin/', admin.site.urls),
]
```

Just change the first line. Tha API views will stay the same, they already are in a different url.py and view.py.